tests/src/e4e.ts

```diff
@@ -98,76 +110,86 @@ Cypress.Commands.add('E4EAddNewConnection', (profileName, host) => {
   });
 });
 
-declare namespace Cypress {
-  interface Chainable<Subject> {
-    /**
-     * Get the connection(profile name)
-     *
-     * @example cy.connectionNameE4E(Chodov);
-     */
-    connectionNameE4E(profileName: string): Chainable<any>;
+declare global {
+  namespace Cypress {
+    interface Chainable<Subject> {
+      /**
+       * Get the connection(profile name)
+       *
+       * @example cy.connectionNameE4E(Chodov);
+       */
+      connectionNameE4E(profileName: string): Chainable<any>;
+    }
   }
 }
 Cypress.Commands.add('connectionNameE4E', (profileName) => {
   cy.get(`[id="/0:${profileName}"]`);
 });
 
-declare namespace Cypress {
-  interface Chainable<Subject> {
-    /**
-     * Click on the connection name(profile)
-     *
-     * @example cy.clickOnConnectionNameE4E(TEST.REPORT.EXAMPLE);
-     */
-    clickOnConnectionNameE4E(profileName: string): Chainable<any>;
+declare global {
+  namespace Cypress {
+    interface Chainable<Subject> {
+      /**
+       * Click on the connection name(profile)
+       *
+       * @example cy.clickOnConnectionNameE4E(TEST.REPORT.EXAMPLE);
+       */
+      clickOnConnectionNameE4E(profileName: string): Chainable<any>;
+    }
   }
 }
 Cypress.Commands.add('clickOnConnectionNameE4E', (profileName) => {
   cy.connectionNameE4E(profileName).trigger('mousedown').click();
 });
 
-declare namespace Cypress {
-  interface Chainable<Subject> {
-    /**
-     * Unock the report
-     *
-     * @example cy.deleteE4EConnection(TEST.REPORT.EXAMPLE);
-     */
-    deleteE4EConnection(profileName: string): Chainable<any>;
+declare global {
+  namespace Cypress {
+    interface Chainable<Subject> {
+      /**
+       * Unock the report
```

Comment:
"Unlock"?

Reply:
This helper's comment is not valid, should be `Delete E4E connection`.